accountClient/api.go

```diff
@@ -66,3 +68,27 @@ type CreateAccountResponse struct {
 type InternalGetClientAccountResponse struct {
 	AccountID string `json:"account_id"`
 }
+
+// ValidateTokenRequest represents a valid request to the account service's auth/validate endpoint.
+type ValidateTokenRequest struct {
+	Token string `json:"token"` //The token to validate
+}
+
+// ValidateTokenResponse represents the result of calling the account service's auth/validate endpoint.
+type ValidateTokenResponse struct {
+	AccountID string `json:"account_id"` //The account ID associated with this token
+	Valid     bool   `json:"valid"`      //Whether the token was valid
+}
+
+// RegTokenInfo is the return from the token endpoint on a valid request
+type RegTokenInfo struct {
+	Token       string
+	AccountID   uuid.UUID `json:"account_id"`
```

Comment:
why not a string like the other AccountID at line 79?

Reply:
@lkwdwrd Preference here?

Reply:
No preference -- let's stay consistent